holoviews/core/data/pandas.py

```diff
@@ -210,7 +215,7 @@ def add_dimension(cls, columns, dimension, dim_pos, values, vdim):
     @classmethod
     def dframe(cls, columns, dimensions):
         if dimensions:
-            return columns.reindex(columns=dimensions)
```

Comment:
This change snuck in and is unrelated, but I've been meaning to commit it for a while anyway.